templates/base/packages/nextjs/package.json

```diff
@@ -20,6 +20,7 @@
     "@uniswap/sdk-core": "~4.0.1",
     "@uniswap/v2-sdk": "~3.0.1",
     "blo": "~1.0.1",
+    "burner-connector": "^0.0.3",
```

Comment:
I guess in this case we don't care so much because we own the lib haha

But, should we use `~` for consistency?

Reply:
Ohh yes yes my bad, udpate it at https://github.com/scaffold-eth/create-eth/pull/15/commits/69fe824fea72ccd748472d32a45e8dab4955dde0